apps/web/lib/services/blockchain/__tests__/mock-graduation-nft.test.ts

```
@@ -0,0 +1,562 @@
+/**
+ * Unit tests for MockGraduationNFTService
+ */
+
+import { describe, test, expect, beforeEach } from 'bun:test';
+import { MockGraduationNFTService } from '../mock-graduation-nft.service';
+import { MockProgressTrackerService } from '../mock-progress-tracker.service';
+import { MockBadgeTrackerService } from '../mock-badge-tracker.service';
+import { MockAuthControllerService } from '../mock-auth-controller.service';
+import { NFTError, BadgeType } from '~/lib/types/blockchain/contract-interfaces.types';
```

Biome (2.1.2)

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)